l2l/optimizers/gridsearch/optimizer.py

```diff
@@ -105,7 +105,7 @@ def __init__(self, traj,
         #: The current generation number
         self.g = 0
         # Expanding the trajectory
-        grouped_params_dict = {key: value for key, value in self.param_list.items()}
+        grouped_params_dict = {'individual.' + key: value for key, value in self.param_list.items()}
```

Comment:
You reintroduced the `individual` key again after removing it in the previous commit. Why?

Reply:
Because I made a mistake: As my individual provided (with create_individual) a dictionary with {key: value}, and I got back another one of the form {"ïndividual.key": value} for the simulate method, I thought it was a mistake. Now I see it works that way (to be able to access things directly as variables, I suppose), so I realize it was MY mistake.
Note: As I am trying to do generic things on my side, and I already was using dictionaries, what I did on my own optimizee was to "clean" the "individual." part of the key string at the simulate function.